proto/internal/temporal/server/api/persistence/v1/namespaces.proto

```diff
@@ -66,3 +67,9 @@ message NamespaceReplicationConfig {
     repeated string clusters = 2;
     temporal.api.enums.v1.ReplicationState state = 3;
 }
+
+// Represents a historical replication status of a Namespace
+message ReplicationStatus {
+    google.protobuf.Timestamp status_time = 1 [(gogoproto.stdtime) = true];
+    string active_cluster_name = 3;
```

Comment:
Instead of keep track of active cluster name, we should use failoverVersion.

Reply:
I agree, except that failoverVersion is not present in any of these APIs, so I would have to change the way this works. Do you think it's important enough to block this PR?

Reply:
see line 41 of this file.

Reply:
Done